GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -1707,13 +1758,107 @@ private void writeStatsAndGraphsSheet(
         }
     }
 
-    /**
-     * creates a cell in the row passed in and sets the style and value (if non-null)
-     */
+    private int writeCollectionStats(Map<QuestionGroupDto,
+            List<QuestionDto>> questionMap,
+            Sheet sheet,
+            int curRow) {
+        final int tagCol = 0;
+        final int valCol = 3;
+        //Global statistics
+        //Calculate them first (due to the window, we cannot sum this while looping and 
+        // then go back up and draw it last)
+        int totalQuestions = 0;
+        for (List<QuestionDto> group : questionMap.values()) {
+            totalQuestions += group.size();
+        }
+        //Now draw them
+        curRow++;
```

Comment:
does curRow stand for currentRow? is there a reason for having these var names a bit shortened?